docs/theory/Validation_pipe_diameter_sizing.rst

```diff
@@ -0,0 +1,1148 @@
+From about DN250 and larger, the pumping energy becomes more or less negligible compared to the heat losses. The results show that the heat losses do not increase monotonically.
+This is because the isolation thicknesses do vary over the different diameters. 
+
+:numref:`V_physics_validation` also shows the optimal solution found by the optimizer routine, namely DN200, which is due to the maximum velocity criterium of 3.0 m/s as :numref:`V_physics_validation_temperature` shows. 
+On the other hand, the WANDA steady-state results show that the least about of energy is needed for DN500. That is because the difference in optimization: energy vs costs. 
+So it depends on how the costs for the extra required energy outweigh the costs of a larger pipe diameter. We investigate that in the next subsection.
+
+.. _V_physics_validation:
+
+.. figure:: ../images/physics_validation.png
+    :figwidth: 6.94792in
+    :align: center
+
+    Required energy [W/m] results for different pipe diameter for the pipes without (left) and with insulation (right).
+
+.. _V_physics_validation_temperature:
+
+.. figure:: ../images/physics_validation_temperature.png
+    :figwidth: 6.94792in
+    :align: center
+
+    (Left) temperature and (right) velocity at the outlet of the supply pipe results for different pipe diameter for the pipes with and without insulation.
+
+
+.. _sec_V_pipe_D_validation_costs:
+
+Cost validation
+^^^^^^^^^^^^^^^
+
+:numref:`fig_physics_validation_costs` shows the the costs for different diameters according to the setup of the optimizer with and without insulation. 
+
+.. _fig_physics_validation_costs:
+
+.. figure:: ../images/physics_validation_costs.png
+    :figwidth: 6.94792in
+    :align: center
+
+    Cost validation of minimizing the pipe diameter case.
+
+Without any insulation, the costs associated with the heat losses are dominant, while those are less dominant for pipes with insulation.
+The costs are mainly determined by the variable OPEX.
+Note that the routine accounts for 25 years of variable opex costs, but the optimization results (including the TCO) only include 1 year of opex costs. The capex costs account for the whole lifetime of the system.
+We apply the same for the WANDA results.
+
+For the simulated cases with WANDA, the smallest simulated diameter DN125 requires the minimum total energy, because the costs associated with the heat losses dominates over the costs associated with pumping. 
+On the other hand, the optimizer routine determines a minimum diameter of DN200 as we have shown before in :numref:`V_physics_validation`. 
+This is due to the minimum velocity criterium applied to the optimization.
+
+Not that we have only considered the demand peak hour. 
+Typically, the cost associated with the heat losses will be at least an order higher in summer when the demand for heat is low, which could affect this cost figure.
+
+
+
+.. _sub_sec_V_ins_classes:
+
```

Comment:
May be worth discussing the implications of this difference in the context of network design

Reply:
Yes to be discussed